scss/components/brand-lockup/_brand-lockup.scss

```diff
@@ -49,22 +49,41 @@ $logo-width: 110px;
 }
 
 %block--lockup__site-name-and-slogan-text {
-  text-shadow: 0 0 5px rgba(0, 0, 0, 0.35);
+  // text-shadow: 0 0 5px rgba(0, 0, 0, 0.35);
   text-transform: uppercase;
   font-size: em(36px);
   font-weight: 600;
   line-height: 1;
   letter-spacing: 2.2px;
   margin: 0 27px 0 23px;
   padding-bottom: 6px;
+  position: relative;
   text-align: justify;
   width: $logo-width;
 
   @include grid-media($header-md) {
     margin: 0 12px 0 8px;
   }
 
+  &::after {
+    content: '';
+    display: block;
+    background-color: rgba(0, 0, 0, .5);
```

Comment:
You are one trixy hobbit. I've tried now with a few images and most I don't notice the slight darkening behind the text.